systems/summit_user_guide.rst

```diff
@@ -2828,7 +2828,11 @@ debugging features (setting breakpoints, stepping through code,
 examining variables), DDT also supports attaching to already-running
 processes and memory debugging. In-depth details of DDT can be found in
 the `Official DDT User
-Guide <https://www.allinea.com/user-guide/forge/userguide.html>`__.
+Guide <https://www.allinea.com/user-guide/forge/userguide.html>`__, and
+instructions for how to use it on OLCF systems can be found on the
+`Forge (DDT/MAP) Software Page <https://www.olcf.ornl.gov/software_package/forge/>`__. DDT is the
```

Comment:
@jack-morrison @isubsmith is this currently place place for this link to point to? If updates are coming later, that's fine too, but I didn't know what is currently in place.